pulsar-io/kinesis/src/main/java/org/apache/pulsar/io/kinesis/BaseKinesisConfig.java

```diff
@@ -35,6 +35,13 @@ public abstract class BaseKinesisConfig implements Serializable {
     )
     private String awsEndpoint = "";
 
+    @FieldDoc(
+            required = false,
+            defaultValue = "",
+            help = "Cloudwatch end-point url. It can be found at "
+                    + "https://docs.aws.amazon.com/general/latest/gr/rande.html"
+    )
+    private String cloudwatchEndpoint = "";
```

Comment:
·cloudWatchEndpoint` ？

Reply:
https://github.com/apache/pulsar/pull/21004/files#diff-8cd0be28b901ed3ce6cebd6613bdc941f9748c0351f370573f1dd8cff32d00b7L130

To be consistent with the previous name.